WangscapeTest/TestCornerCombiner.cpp

```diff
@@ -1,29 +1,30 @@
 #include <gtest/gtest.h>
 
 #include <noise/module/CornerCombinerBase.h>
-#include <noise/module/ModuleFactories.h>
+#include <noise/module/ReseedableOps.h>
 
 class TestCornerCombiner : public ::testing::Test
 {
 protected:
     noise::module::CornerCombinerBase cc1;
     noise::module::CornerCombinerBase cc05;
     noise::module::CornerCombinerBase cc2;
-    noise::Reseedable ccg2_p;
+    noise::module::ReseedablePtr ccg2_p;
     TestCornerCombiner():
-        cc1(), cc05(0.5), cc2(2.),
         ccg2_p(noise::module::makeCornerCombiner(true, true))
     {
+        cc05.SetPower(0.5);
+        cc2.SetPower(2.);
```

Comment:
`CornerCombiner` needs to be tested with `Power` parameters `< 1`, `== 1`, and `> 1`. These are just example parameters for which the expected output values can be easily calculated; they don't have any special meaning.